[2018-09-05 17:34:17] php.INFO: User Deprecated: Referencing controllers with a single colon is deprecated since Symfony 4.1, use "fos_oauth_server.controller.token::tokenAction" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Referencing controllers with a single colon is deprecated since Symfony 4.1, use \"fos_oauth_server.controller.token::tokenAction\" instead. at /var/www/html/sgf/novosga/vendor/symfony/framework-bundle/Routing/DelegatingLoader.php:98)"} []
[2018-09-05 17:34:17] console.ERROR: Error thrown while running command "cache:warmup --env=prod". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:113, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:50, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:46)","command":"cache:warmup --env=prod","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2018-09-05 17:34:17] console.DEBUG: Command "cache:warmup --env=prod" exited with code "1" {"command":"cache:warmup --env=prod","code":1} []
[2018-09-05 17:35:27] console.ERROR: Error thrown while running command "cache:warmup --env=prod". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:113, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:50, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:46)","command":"cache:warmup --env=prod","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2018-09-05 17:35:27] console.DEBUG: Command "cache:warmup --env=prod" exited with code "1" {"command":"cache:warmup --env=prod","code":1} []
[2018-09-05 17:36:54] console.ERROR: Error thrown while running command "cache:warmup --env=prod". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:113, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:50, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:46)","command":"cache:warmup --env=prod","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2018-09-05 17:36:54] console.DEBUG: Command "cache:warmup --env=prod" exited with code "1" {"command":"cache:warmup --env=prod","code":1} []
[2018-09-06 08:37:39] php.INFO: User Deprecated: Referencing controllers with a single colon is deprecated since Symfony 4.1, use "fos_oauth_server.controller.token::tokenAction" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Referencing controllers with a single colon is deprecated since Symfony 4.1, use \"fos_oauth_server.controller.token::tokenAction\" instead. at /var/www/html/sgf/novosga/vendor/symfony/framework-bundle/Routing/DelegatingLoader.php:98)"} []
[2018-09-06 08:37:40] console.ERROR: Error thrown while running command "cache:warmup --env=prod". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:113, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:50, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:46)","command":"cache:warmup --env=prod","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2018-09-06 08:37:40] console.DEBUG: Command "cache:warmup --env=prod" exited with code "1" {"command":"cache:warmup --env=prod","code":1} []
[2018-09-06 08:38:18] console.ERROR: Error thrown while running command "cache:warmup --env=prod". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:113, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:50, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:46)","command":"cache:warmup --env=prod","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2018-09-06 08:38:18] console.DEBUG: Command "cache:warmup --env=prod" exited with code "1" {"command":"cache:warmup --env=prod","code":1} []
[2018-09-06 08:38:48] console.ERROR: Error thrown while running command "cache:warmup --env=prod". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:113, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:50, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:46)","command":"cache:warmup --env=prod","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2018-09-06 08:38:48] console.DEBUG: Command "cache:warmup --env=prod" exited with code "1" {"command":"cache:warmup --env=prod","code":1} []
[2018-09-06 08:39:33] console.ERROR: Error thrown while running command "cache:warmup --env=prod". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:113, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:50, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:46)","command":"cache:warmup --env=prod","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2018-09-06 08:39:33] console.DEBUG: Command "cache:warmup --env=prod" exited with code "1" {"command":"cache:warmup --env=prod","code":1} []
[2018-09-06 08:41:57] console.ERROR: Error thrown while running command "cache:warmup --env=prod". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:113, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:50, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/html/sgf/novosga/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:46)","command":"cache:warmup --env=prod","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2018-09-06 08:41:57] console.DEBUG: Command "cache:warmup --env=prod" exited with code "1" {"command":"cache:warmup --env=prod","code":1} []
